### PR TITLE
support role graphs that contain admin_option

### PR DESCRIPTION
**Pull request by apotterri**

### CODE REVIEW

File: jenkins.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ if [ -z "$CONJUR_CONTAINER" ]; then
 	    docker pull $DOCKER_IMAGE
 	fi
 	
-	cid=$(docker run -d -v ${PWD}:/src/conjur-api $DOCKER_IMAGE)
+	cid=$(docker run --privileged -d -v ${PWD}:/src/conjur-api $DOCKER_IMAGE)
```
Comment (kgilpin) [inline jenkins.sh]: Why privileged? I haven't seen this before.

File: lib/conjur/graph.rb [hunk 2]
```diff
@@ -248,20 +254,20 @@ def as_json short = false
...
       def to_a
-        [@parent, @child]
+        [@parent, @child].tap {|a| a.push(@admin_option) unless @admin_option.nil?}
       end
 
       # @api private
       # :nodoc:
       def to_s
-        "<Edge #{parent.id} --> #{child.id}>"
+        "<Edge #{parent.id} --> #{child.id} (admin? #{@admin_option})>"
```
Comment (kgilpin) [inline lib/conjur/graph.rb]: "admin?" may not be defined

File: jenkins.sh (hunk 1 above)
Comment (apotterri) [inline jenkins.sh]: When I try to run without `--privileged`, I get this:
```
Alans-MacBook-Pro:conjur ajp$ docker -v
Docker version 1.12.0-rc2, build 906eacd, experimental
Alans-MacBook-Pro:conjur ajp$ docker run -i registry.tld/conjur-appliance-cuke-master:4.8-stable
keyctl_join_session_keyring: Operation not permitted
```
The alternative is to use a seccomp profile, like the one Rafal created: https://github.com/conjurinc/appliance/blob/master/distrib/seccomp.json, or disable it altogether with `--security-opt seccomp:unconfined`.

Reading a little more about `--privileged`, it's probably too big a hammer here. Given that this is just for running tests, though, maybe I'll just switch it to `--security-opt seccomp:unconfined`.

File: lib/conjur/graph.rb (hunk 2 above)
Comment (apotterri) [inline lib/conjur/graph.rb]: It's just a literal, not a reference to a variable.

File: jenkins.sh (hunk 1 above)
Comment (apotterri) [inline jenkins.sh]: Except that docker v1.9 (that's running on jenkins) doesn't support it. I'll stick with `--privileged`.

**Pull request merged**